packages/python/readme_metrics/flask_readme.py

```diff
@@ -39,6 +42,9 @@ def before_request(self):
             self.config.LOGGER.exception(e)
 
     def after_request(self, response):
+        if request.method == "OPTIONS":
```

Comment:
Is it possible to get here in the request lifecycle when it's an `OPTIONS` call?

Reply:
@gratcliff @erunion i've added some tests for this feature and made some minor changes